docusaurus/docs/reference/config-types/host.v1.md

```diff
@@ -0,0 +1,263 @@
+sidebar_position: 15
+---
+
+# The `host.v1` Config Type
+
+The `host.v1` configuration type defines how edge routers or tunnelers can make outgoing connections
+for associated services. The config type also allows specifying health checks for the service. An
+endpoint in a `host. v1` config can be specified explicitly or can be configured to use information
+passed through from a tunneler intercepting traffic, allowing it to forward traffic.
+
+The `host. v1` type only allows defining a single host endpoint. If multiple endpoints are required,
+use the [`host. v2`](./host.v2.md) config type instead.
+
+## Endpoint Configuration
+
+For services with defined endpoints, the following three attributes are required:
+
+* `address` - an IP or DNS address to connect to
+* `port` - the network port to connect to
+* `protocol` - tcp or udp
+
+**Examples**
+
+This service is hosted by one application server.
+
+```json
+{
+  "address": "192.168.100.1",
+  "port": 80,
+  "protocol": "tcp"
+}
+```
+
+## Forwarding Configuration
+
+Services that are forwarding traffic from an OpenZiti tunneling proxy may use the following
+properties to indicate what should be forwarded:
+
+* `forwardProtocol` - flag indicating that the protocol of the forwarded connection is to be used.
+  Can only be set to true.
+* `allowedProtocols` - the list of allowed protocols. Valid values include `tcp` and `udp`
+* `forwardAddress` - flag indicating that the target address of the forwarded connection is to be
+  used. Can only be set to true.
+* `allowedAddresses` - the list of allowed addresses. Valid values include IPs, hostnames and CIDRs
+* `forwardPort` - flag indicating that the target port of the forwarded connection is to be used.
+  Can only be set to true.
+* `allowedPortRanges` - the list of allowed port ranges.
+    * Example: `allowedPortRanges: [ {"low" : 80, "high" : 80}, {"low" : 8080, "high" : 8090} ]`
+* `allowedSourceAddresses` - list of addresses in IP, hostname or CIDR
+    * hosting tunnelers establish local routes for the specified source addresses so binding will
+      succeed
+
+**Examples**
+
+```json
+{
+  "forwardAddress": true,
+  "allowedAddresses": [
+    "192.168.1.0/24",
+    "10.0.0.1/16"
+  ],
+  "forwardPort": true,
+  "allowedPorts": [
+    {
+      "low": "1024",
+      "high": 2048
+    }
+  ],
+  "forwardProtocol": true,
+  "allowedProtocols": [
+    "tcp",
+    "udp"
+  ]
+}
+```
+
+Note that not everything must be forwarded. For example the address is not forwarded in the example
+below. The port and protocol are forwarded and the corresponding 'allow' is set, but the address is
+statically set to '192.168.100.1
+
+```json
+{
+  "address": "192.168.100.1",
+  "forwardPort": true,
+  "allowedPorts": [
+    {
+      "low": "1024",
+      "high": 2048
+    }
+  ],
+  "forwardProtocol": true,
+  "allowedProtocols": [
+    "tcp",
+    "udp"
+  ]
+}
+```
+
+Health checks and listen options also can be specified for each terminator.
```

Comment:
this is the first use of _terminator_. Maybe an association between host.vX and terminator is made elsewhere, but I didn't see it. I think it would be better to just mention `host.vX` here. e.g. "... also can be specified in host.v1/host.v2 configurations".

Reply:
I removed references to `terminator` for now. Once we have a entity reference section, I think we can link to that and explain how they tie together.